backend/blueprint/api/admin/users.py

```diff
@@ -0,0 +1,254 @@
+from flask import Blueprint, jsonify, request
+from api.utils.response import Response
+from firebase_admin import auth
+
+
+router = Blueprint('users', __name__)
+
+
+@router.route('/createuser', methods=['POST'])
+def create_user():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(Response(500).generateMessage(str(e)))
+
+    try:
+        user = auth.create_user(
+            display_name=data['name'],
+            email=data['email'],
+            email_verified=data['email_verified'],
+            photo_url=data['photo_url'],
+            password=data['password'],
+            disabled=data['disabled']
+        )
+        return jsonify({
+            'message': 'Sucessfully created new user named {0} with user-id: {1}'.format(user.display_name, user.uid),
+            'status': 200
+        })
+    except Exception as e:
+        print(e)
+        return jsonify({
+            'message': str(e),
+            'status': 400
+        })
+
+
+@router.route('/updateuser', methods=['POST'])
+def update_user():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(Response(500).generateMessage(str(e)))
+
+    try:
+        user = auth.update_user(
+            data['uid'],
+            display_name=data['name'],
+            email=data['email'],
+            email_verified=data['email_verified'],
+            photo_url=data['photo_url'],
+            password=data['password'],
+            disabled=data['disabled']
+        )
+        return jsonify({
+            'message': 'Sucessfully updated user: {0}'.format(user.uid),
+            'status': 200
+        })
+    except Exception as e:
+        return jsonify({
+            'message': str(e),
+            'status': 400
+        })
+
+
+@router.route('/deleteuser', methods=['POST'])
+def delete_user():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(Response(500).generateMessage(str(e)))
+
+    try:
+        auth.delete_user(
+            data['uid']
+        )
+        return jsonify({
+            'message': 'Successfully deleted user',
+            'status': 200
+        })
+    except Exception as e:
+        return jsonify({
+            'message': str(e),
+            'status': 400
+        })
+
+
+@router.route('/user/<uid>/userdetails', methods=['GET'])
+def get_user_details(uid):
+    try:
+        user_details = auth.get_user(uid)
+        name = user_details.display_name
+        email = [{'address': user_details.email, 'verified': user_details.email_verified}]
+        photo_url = user_details.photo_url
+        provider_id = user_details.provider_id
+        disabled = user_details.disabled
+        custom_claims = user_details.custom_claims
+
+        return jsonify({
```

Comment:
we need to update Response class to hold admin user details. Since we are maintaining a dedicated module for handling responses we should follow the same trend here too.

Reply:
ok, I will update `Response` class

Reply:
@djmgit handling responses through a dedicated module will be difficult as have to handle multiple cases like returning normal message, json, etc. Should we go the normal way like using a pre-written function like jsonify?

Reply:
@vaibhavsingh97, Can you please elaborate on this how it is difficult to use Response class?

Reply:
@gabru-md @ParthS007 
right now, In https://github.com/fossasia/badgeyay/blob/development/backend/blueprint/api/utils/response.py#L12:L16 @yashLadha had customized the function. Here, If I want to pass general response like "User created successfully", other fields will also appear like `id`, `attributes` which are not needed, if we want any dynamic response like in this PR,  like `Successfully created new user named {0} with user-id: {1}'.format(user.display_name, user.uid)` then we need to modify the method. So, in the end, we will end up add possible cases.

Reply:
first of all, there is **no correlation with the dynamic data and response class**.
>if we want any dynamic response like in this PR, like Successfully created new user named {0} with user-id: {1}'.format(user.display_name, user.uid) then we need to modify the method

You will have to modify the method in case of static data as well. :/
 and for the fact you have stated, you can always make new functions. Or just like @yashLadha you can also modify the function so that it can be used elsewhere as well. It is quite simple.

There are changes you can do so that they comply with your changes and do not break other's work as well. Please see to it.